src/ui/public/visualize/visualize.less

```diff
@@ -16,7 +16,7 @@ visualize {
     display: flex;
     flex-direction: row;
 
-    flex: 1 0;
+    flex: 1 1 0;
```

Comment:
OK, so I think i have an understanding of this problem. To break it down:

1. It's a known bug that IE11 ignores unitless flex-basis values in flex shorthand declarations ([reference on flexbugs](https://github.com/philipwalton/flexbugs#4-flex-shorthand-declarations-with-unitless-flex-basis-values-are-ignored)).

2. The recommended solution to this bug is to use 0%, e.g. `flex: 1 1 0%`.

3. However, Firefox has a bug where it doesn't respect 0% as it's supposed to ([reference on flexbugs](https://github.com/philipwalton/flexbugs/issues/25)).

So, I think the first thing we should do is double-check the latest version of Firefox and make sure that the bug in point 3 still exists. It's possible I reproduced this bug using an older version of Firefox.

If the bug has been fixed, then we're fine using the 0% solution.

If not, then let's explore using an [IE11](http://stackoverflow.com/questions/20541306/how-to-write-a-css-hack-for-ie-11) [CSS Hack](http://stackoverflow.com/questions/18907131/detecting-ie11-using-css-capability-feature-detection/27315792#27315792) to introduce the 0% property after the regular property.